set.mm

```diff
@@ -424242,12 +424242,13 @@ allows us to easily use the theorem in places where (in traditional
        forms of the same theorem (e.g., ~ pm2.43d ).  The labels for inferences
        usually have the suffix "i" (e.g., ~ pm2.43i ).  The labels of theorems
        in "closed form" would have no special suffix (e.g., ~ pm2.43 ) or, if
-       the standard label for the closed form is already in use, suffix "t"
-       (for "theorem" or "tautology", e.g., ~ ancomst or ~ nfimt ).  When an
-       inference is converted to a theorem by eliminating an "is a set"
-       hypothesis, we sometimes suffix the closed form with "g" (for "more
-       general") as in ~ uniex versus ~ uniexg .  In this case, the inference
-       often has no suffix "i".
+       the non-suffixed label is already used, then we add the suffix "t" (for
+       "theorem" or "tautology", e.g., ~ ancomst or ~ nfimt ).  When an
+       inference with an "is a set" hypothesis (e.g., ` A e. _V `) is converted
```

Comment:
I was going to say: the theorem is not "converted", we just add another one; but then, I remembered that sometimes we replaced the old one with the new one, so we can speak of a "conversion" indeed.